webapp/src/main/webapp/skin/less/news.less

```diff
@@ -5,6 +5,10 @@
 @colorIconPrimary: @primaryColor;
 @color-appNews-information: #676767;
 
+//Alerts color
+@primaryColorDefault: #578dc9;
```

Comment:
it already exists as a variable in variables.less , why don't you use it directly,?